src/config/config.go

```diff
@@ -64,7 +64,7 @@ func LoadConfig(configPath string) *Config {
 			Mode:          ModeBytes,
 			BytesPerSlice: 524_288_000, // 500 MiB
 			RowsPerSlice:  1_000_000,
-			Gzip:          false,
+			Gzip:          true,
```

Comment:
Robil som este nejake testy, a pri gzip kompresii job vzdy dobehne rychlejsie. Po poslednych optimalizaciach je rozdiel este vacsi:

![image](https://user-images.githubusercontent.com/19371734/118238691-26ddd400-b499-11eb-8435-89d4e16522b0.png)

Preto chcem gzip by default zapnut. Ak by to nieto nechcel, tak si to vypne.